### ISSUE TITLE
Don't allow generated UUIDs in Single File type

### ISSUE THREAD

**Issue by JLueke** [directus/directus#3241]
1. Create a collection with a field "created_by" of type Single File and set it up to save the current users id on creation.
2. Create an Item
3. Open list view of collection

What I expected to happend:
I see some sort of user display in the column "created_by".

What does happen:
I see "--" in the column "created_by".

Neither fields with special user-created nor user-updated are shown correctly in list view.
If you open the filter you get a list of file relate properties.

Tested with official Docker Image v9.0.0-rc.17 and v9.0.0-rc.19.
There are some differences between those versions regarding missing Options for field creation, but show the same bug.

![user is handled like file](https://user-images.githubusercontent.com/17501454/100551922-1c7da400-3284-11eb-8dc5-ef18774165a7.png)

**Comment by caiguofeng0209** [comment]
👁👁👁👁

**Comment by JLueke** [comment]
Futhermore I noticed, that the user is displayed correctly in the list of the activity log.
I'll check the database later in how the user field here differs from the one created via the UI.

**Comment by rijkvanzanten** [comment]
Just out of curiosity, why did you choose `Single File` as the type for a _user_? 

To me, the problem here is that we allow to auto-set / generate the UUID for File types. That won't work, as the file type is meant to link to files in directus_files 🙂

**Comment by JLueke** [comment]
> Just out of curiosity, why did you choose `Single File` as the type for a _user_?
> 
> To me, the problem here is that we allow to auto-set / generate the UUID for File types. That won't work, as the file type is meant to link to files in directus_files 🙂

because there is no separate user interface and the single file type allowed updating the user. The display of the user worked in the detailed view. So I thougth it was intentionally to use `Single File` in this case.

**Comment by JLueke** [comment]
The directus_activity fields aren't defined in directus_fields table. Guess we'll need to search the frontend to solve this.

**Comment by JLueke** [comment]
@rijkvanzanten how do I set the user field correctly, if I wan't it to be set automatically?

Now I tested it with the standard field with type UUID. I works in the detailed vue but not in the list, since the relation is to directus_users is not set. If I add the relation manually in the database, it is shown correctly in the list view.

I guess, there's no proper field form implemented to create a user field yet. Is that correct?
